doc/user_manual/database_data.tex

```diff
@@ -148,6 +148,10 @@ \section{DataObjects}
    \begin{itemize}
      \item \xmlAttr{var}, \xmlDesc{required string attribute}, the dimension name of this index (e.g. time)
    \end{itemize}
+   This XML node allows the following attribute:
+   \begin{itemize}
+     \item \xmlAttr{autogenerate}, \xmlDesc{optional boolean attribute}, if this index should be generated automatically if it does not exist.
```

Comment:
Could you add more detail on this feature?

Reply:
Added more detail.